README.adoc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 IBM Corporation and others.
+// Copyright (c) 2018 IBM Corporation and others.
```

Comment:
If you're updating an existing file, the copyright year should be `2017, 2018`, you can just change it to `2018`.